### PR TITLE
PRM: Make iron bars more common

### PR DESCRIPTION
**Pull request by pisskop**
Iron bars, i.e. solid iron rods i.e. improvised weapon.

Makes it drop from guardrails

### CODE REVIEW

File: data/mods/PKs_Rebalance/pk_terrain.json [hunk 1]
```diff
@@ -0,0 +1,108 @@
+[
+    {
+        "type" : "terrain",
+        "id" : "t_guardrail_bg_dp",
+        "edit-mode": "modify",
+        "name": "guard rail",
+        "symbol": "#",
+        "color": "ltgray",
+        "move_cost": 3,
+        "flags": ["TRANSPARENT", "NOITEM", "REDUCE_SCENT", "MOUNTABLE", "SHORT", "THINOBSTACLE", "ROAD"],
```
Comment (illi-kun) [inline data/mods/PKs_Rebalance/pk_terrain.json]: ```THINOBSTACLE``` should be ```THIN_OBSTACLE``` (at least per ```JSON_FLAGS.md``` data).

Comment (mugling) [inline data/mods/PKs_Rebalance/pk_terrain.json]: @illi-kun once #18349 is done we can work on extending the linter to check flags

Comment (illi-kun) [inline data/mods/PKs_Rebalance/pk_terrain.json]: @mugling sure. BTW, previously I did similar flags check using bash script with ```JSON_FLAGS.md``` as a dictionary but your implementation should be more robust and reliable, I guess.

Comment (mugling) [inline data/mods/PKs_Rebalance/pk_terrain.json]: I want to try and handle conflicting flags and the like. If you want to help extend the ruleset help would be appreciated.

Comment (illi-kun) [inline data/mods/PKs_Rebalance/pk_terrain.json]: I'm going to spend some time tomorrow for reading more info about your recent activity (related to lint & json files), so I'll let you know if I have something to add to your PRs.

**Comment by pisskop** [comment]
this pr is quite literally a copy of the guardrail already ingame, with a tag 'edit-mode' and a chance to drop a metal rod instead of steel odds and ends  :|

**Comment by illi-kun** [comment]
Sorry for side discussion in your PR. Please fix the incorrect flag and it's good to go.

**Pull request merged**